docs/docusaurus/docs/contributing/application_integration_support.md

```diff
@@ -0,0 +1,100 @@
+
+## Data stores
+
+The following table defines the data stores supported by GX and the community.
+
+| GX supported                             | Community supported                                                |
+|------------------------------------------|--------------------------------------------------------------------|
+| <i class="fas fa-check-circle" style={{color: "#28a745"}}></i> AWS S3<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> GCS Buckets     | <i class="fas fa-check-circle" style={{color: "#28a745"}}></i> Azure Blob Store              | 
+
+## Checkpoint integrations
+
+The following table defines the Checkpoint integrations supported by GX and the community.
+
+| GX supported                             | Community supported                                                |
+|------------------------------------------|--------------------------------------------------------------------|
+| <i class="fas fa-check-circle" style={{color: "#28a745"}}></i> Slack<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> Email     | <i class="fas fa-check-circle" style={{color: "#28a745"}}></i> MS Teams<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> Pager Duty<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> Opsgenie<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> Amazon SNS<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> General API<br/><i class="fas fa-circle" style={{color: "#dc3545"}}></i> DataHub        | 
+
+## Credential stores
+
+The following table defines the credential stores supported by GX and the community.
+
+| GX supported                             | Community supported                                                |
+|------------------------------------------|--------------------------------------------------------------------|
+| <i class="fas fa-check-circle" style={{color: "#28a745"}}></i> YAML<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> AWS Secrets Store<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> Azure Blob Store<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> GCP Secrets Manager<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> Azure Key Vault<br/><i class="fas fa-check-circle" style={{color: "#28a745"}}></i> Databricks Secrets Store<br/><i class="fas fa-circle" style={{color: "#dc3545"}}></i> HashiCorp Vault    |               |
```

Comment:
I'm thinking HashiCorp shouldn't be under GX supported.

Reply:
Also uncertain that all of these are under test -- certainly we don't have example scripts showing these under test for Databricks Secrets Store or Azure Blob Store.